debian/control

```diff
@@ -2,58 +2,56 @@ Source: swtpm
 Maintainer: Stefan Berger <stefanb@linux.vnet.ibm.com>
 Section: misc
 Priority: optional
-Build-Depends: automake,
-	       autoconf,
-	       libtool,
-	       debhelper (>= 9),
```

Comment:
The `debhelper` dependency version should've been bumped to 10 when `debian/compat` was changed from 9 to 10 in [this commit](https://github.com/stefanberger/swtpm/commit/d321bfa6b4f9264f9bef8caba7ef71470e00c1c2#diff-3e7fb53c4ba564e5ffb9bad1a17b2fa94f3125302a125fa8e17d7b73d6b728ac).